### ISSUE TITLE
Maintain single spec file

### ISSUE THREAD

**Issue by aweiteka** [projectatomic/nulecule#32]
Currently we have both a human-readable README and a machine-readable JSON file to describe the specification. With some extension we could put all of the additional data from the README into the JSON and create a parser script to render it.

**Comment by aweiteka** [comment]
Approach: add `--format` option to `compose.py` https://github.com/projectatomic/nulecule/blob/master/spec/0.0.1-alpha/compose/compose.py

```
$ python compose.py schema.json --output README.asciidoc --format asciidoc
```

For completeness we also need to add some arbitrary text to the schema file such as [Introduction](https://github.com/projectatomic/nulecule/blob/master/spec/0.0.1-alpha/README.md#introduction) and [Data types](https://github.com/projectatomic/nulecule/blob/master/spec/0.0.1-alpha/README.md#data-types).

**Comment by vpavlin** [comment]
We will probably need either extend `description` in schema or introduce new keys like `doc_text`...any thoughts about preferred approach?

**Comment by aweiteka** [comment]
Good idea. I think `doc_text` is best alongside some top-level data for Intro text, etc.

**Comment by aweiteka** [comment]
Consider https://github.com/swagger-api/swagger-spec
